go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/edwarnicke/serialize v1.0.4
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/networkservicemesh/api v0.0.0-20201015041133-d89219eefcc1
-	github.com/networkservicemesh/sdk v0.0.0-20201018180237-f46b09458475
```

Comment:
Could you also downgrade google.golang.org/grpc to v1.32.0?

Reply:
Done